packages/shared/src/components/onboarding/FilterOnboardingV4.tsx

```diff
@@ -63,6 +65,7 @@ export function FilterOnboardingV4({
     feedId,
     shouldFilterLocally,
   });
+  const shouldShuffleTags = useFeature(feature.onboardingShuffleTags);
```

Comment:
This file is also used for custom feeds, we need to ensure those users don't get enrolled/evaluated or see this.

Reply:
Maybe we can evaluate it on `EditTag` component level?

Reply:
Yeah, could probably move it up in the tree to onboarding specific. Or use router to check if path is onboarding